src/main/features/ccj/routes/repayment-plan.ts

```diff
@@ -15,7 +15,8 @@ function renderView (form: Form<PaidAmount>, res: express.Response): void {
 
   res.render(Paths.repaymentPlanPage.associatedView, {
     form: form,
-    remainingAmount: user.claim.totalAmount - alreadyPaid
+    remainingAmount: user.claim.totalAmountTillToday - alreadyPaid,
+    claim: user.claim
```

Comment:
Do we need to pass claim there? I cannot see template changes, is that something we forgot earlier?

Reply:
I think we have to show the correct amount on the check and send page for CCJ, but this line was changed due to the refactor of totalAmount function to be renamed and refactored to be called totalAmountTillToday

Reply:
Not, check the `features/ccj/views/repayment-plan.njk`. There is no need to pass `claim`. Do I miss something?

Reply:
Ok I’ll check it

Reply:
Just checked with Christine and yes we don't have to pass claim here

Reply:
So I will remove, sorry about the confusion earlier